internal/k8s/watch_test.go

```diff
@@ -259,6 +264,11 @@ func (tf *watchTestFixture) runServices(input []runtime.Object, expectedOutput [
 	assert.Equal(tf.t, expectedOutput, observedServices)
 }
 
+func (tf *watchTestFixture) watchEverythingExpectError(expectedErr string) {
+	_, err := tf.kCli.WatchEverything(tf.ctx, []model.LabelPair{})
```

Comment:
I'm not sure I get why this errors -- can you add a comment about why?

Reply:
Sure!